scripts/upgrade

```diff
@@ -80,6 +87,34 @@ ynh_abort_if_errors
 
 #=================================================
 # STANDARD UPGRADE STEPS
+#=================================================
+# OPEN PORTS
+#=================================================
+
+if [ $discovery -eq 1 ]; then
+        ynh_script_progression --message="Configuring firewall..." --weight=1
+
+        # Open port 1900 for service auto-discovery
+        if ynh_port_available --port=$discovery_service_port; then
+                discovery_service=1
+                ynh_exec_warn_less yunohost firewall allow UDP $discovery_service_port
+        else
+                discovery_service=0
+                ynh_print_warn --message="Port $discovery_service_port (for service auto-discovery) is not available. Continuing nonetheless."
+        fi
+        ynh_app_setting_set --app=$app --key=discovery_service --value=$discovery_service
+
+        # Open port 7359 for client auto-discovery
+        if ynh_port_available --port=$discovery_client_port; then
+                discovery_client=1
+                ynh_exec_warn_less yunohost firewall allow UDP $discovery_client_port
+        else
+                discovery_client=0
+                ynh_print_warn --message="Port $discovery_client_port (for client auto-discovery) is not available. Continuing nonetheless."
+        fi
+        ynh_app_setting_set --app=$app --key=discovery_service --value=$discovery_client
```

Comment:
```suggestion
        ynh_app_setting_set --app=$app --key=discovery_client --value=$discovery_client
```

Reply:
Damn I am dense. Do not code while tired, children.